open-machine-learning-jupyter-book/assignments/ml-advanced/ensemble-learning/random-forests-intro-and-regression.ipynb

```diff
@@ -0,0 +1,1325 @@
+{
```

Comment:
<p>The rubric table is not rendering. </p><p> </p><p>Seems that this Rubric section can be removed safely. It doesn't serve us.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
II have removed it